onnx/test/reference_evaluator_backend_test.py

```diff
@@ -615,6 +616,13 @@ def common_test_onnx_test_run(
                 print("CHECK RUNTIME onnxruntime")
                 from onnxruntime import InferenceSession
 
+                # The new IR version is not supported by onnxruntime yet
```

Comment:
There should be issue with the opset as well. I used onnxruntime to check the reference implementation was giving the same answer. It was not meant to be run on CI but only locally. I'm wondering if we should move those tests inside another subfolder, not included in the onnx package.

Reply:
> There should be issue with the opset as well. 


Thanks for the reminder. I added check for opset as well.

> It was not meant to be run on CI but only locally. I'm wondering if we should move those tests inside another subfolder, not included in the onnx package.

IIUC, python files under onnx/test are not packaged in the onnx Python wheel. If that is the case, it seems fine to have them here? Still, putting them in another directory might be more organized (remind users these relevant tests need onnxruntime).